src/js/components/app.coffee

```diff
@@ -96,19 +101,26 @@ App = React.createClass
         sequence: sequence
         pageId: firstPage.id
 
-    resources = "activities"
-    id = @state.activityId
-    if @state.sequenceId
-      resources = "sequences"
-      id = @state.sequenceId
-    url = "#{@state.laraBaseUrl}/#{resources}/#{id}/dashboard_toc"
-    $.ajax
-      url: url
-      dataType: "jsonp"
-      success: setSequence
+    if @useFakeData()
+      utils.fakeAjax =>
+      data = sequenceFakeData(@state.activityId)
```

Comment:
I think you missed an indent here.  The data = ... will not be called in the fakeAjax callback